NewSpring/ApollosProject
# [DEV-3689] Adds ability to delete Prayer Request

## DESCRIPTION

As a User, I would like to be able to delete Prayer Requests that I have submitted.

### What does this PR do, or why is it needed?

This PR allows a user to delete a prayer request through the UI. It implements `react-native-action-sheet` which is a new package to take note of. This also adds some tests to the `UserPrayerList` component.

### How do I test this PR?

1. Make sure you have some prayers in `alpha-rock`.
2. yarn
3. yarn start
4. Navigate to the Prayer Menu, then to the `My Prayers` tab.
5. Then select the `...` in the corner of a prayer card and delete!

You can run this query to test to make sure what is in the UI is what's on the server.
```
query {
  getCurrentPersonPrayerRequests {
    id
    firstName
    lastName
    text
    enteredDateTime
  }
}

```

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3689](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3689)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
